merqube_client_lib/pydantic_types.py

```diff
@@ -445,15 +445,22 @@ class PortfolioUom(Enum):
     SHARES = "SHARES"
 
 
-class IdentifierType(Enum):
+class EquityIdentifierType(Enum):
     RIC = "RIC"
     CURRENCY_CODE = "CURRENCY_CODE"
+    SEDOL = "SEDOL"
+    FSYM_ID = "FSYM_ID"
+    TICKER = "TICKER"
+    BBG_TICKER = "BBG_TICKER"
+    CUSIP = "CUSIP"
 
 
 class AssetType(Enum):
     CASH = "CASH"
     THIRD_PARTY_INDEX = "THIRD_PARTY_INDEX"
     EQUITY = "EQUITY"
+    MUTUAL_FUND = "MUTUAL_FUND"
```

Comment:
@aschmied these are now available to clients

Reply:
Is this fully supported yet? I thought there was still more work to be done.

Reply:
Thats a question for @aschmied - this type file is automatically generated from the API, so they will show up here regardless of backend support.